lms/templates/student_account/password_reset.underscore

```diff
@@ -13,7 +13,7 @@
 
     <p class="action-label"><%- gettext("Please enter your email address below and we will send you instructions for setting a new password.") %></p>
 
-    <%= fields %>
+    <%= HtmlUtils.ensureHtml(fields) %>
```

Comment:
Since you're changing all of these, I'm tempted to suggest that you rename the context parameter to ``fieldsHtml``. It is a little confusing to output a file called ``fields`` which sounds like an array of field information directly into the DOM.